custom/icds_reports/migrations/0194_auto_20200528_1024.py

```diff
@@ -0,0 +1,137 @@
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='lw_thr_0_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='lw_thr_15_20_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='lw_thr_1_7_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='lw_thr_21_24_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='lw_thr_21_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='lw_thr_25_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='lw_thr_8_14_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='lw_thr_eligible',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='pw_thr_0_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='pw_thr_15_20_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='pw_thr_1_7_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='pw_thr_21_24_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='pw_thr_21_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='pw_thr_25_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='pw_thr_8_14_days',
+            field=models.IntegerField(null=True),
+        ),
+        migrations.AddField(
+            model_name='aggservicedeliveryreport',
+            name='pw_thr_eligible',
+            field=models.IntegerField(null=True),
+        )
+    ]
+    operations.extend(get_view_migrations())
```

Comment:
Since, we are adding new field, we need to remove this from the previous migration files??